src/test/datascience/debugger.vscode.test.ts

```diff
@@ -81,9 +85,54 @@ suite('VSCode Notebook - Run By Line', function () {
     // Cleanup after suite is finished
     suiteTeardown(() => closeNotebooksAndCleanUpAfterTests(disposables));
 
+    test('Delete temp debugging files', async function () {
+        let tempWatcher;
+        let folderName;
+        try {
+            tempWatcher = fs.watch(os.tmpdir(), (_event, filename) => {
```

Comment:
I'm a smidge iffy on how stable file watching might be. Was solid on my local machine so will see how it does on CI. It's just a fire and forget delete in product code, so I'd prefer not to horse around with product code if this works in a stable fashion.

Reply:
Looks like only an unrelated failure or two on two runs, so moving out of draft.